src/api/format-description.md

```diff
@@ -51,17 +51,47 @@ to have both consecutive literals and consecutive components. Components must be
 between brackets with optional whitespace. If a literal `[` is desired, it must be escaped with a
 second bracket. A closing bracket need not be escaped if alone.
 
-Follows is the syntax for all components. Any of the following may be used where _component_ is
-present in the above diagram.
+## Examples
 
-- **Day of month**
+Time format descriptions can be created at compile-time using `time::macros::format_description!`;
+code will not compile if parsing of the format fails. Alternately,
+
+To parse an `OffsetDateTime` from a string like `Thu, 06 Jan 2022 02:53:35 00:00`, we can use the
+following format description:
+
+```rust
+use time::macros::format_description;
+
+static EXPIRES_TIME_FORMAT: &'static [FormatItem<'static>] = format_description!(
+    "[weekday repr:short], [day padding:zero] [month repr:short] [year repr:full] \
+        [hour repr:24 padding:zero]:[minute padding:zero]:[second padding:zero] \
+        [offset_hour padding:zero]:[offset_minute padding:zero]"
```

Comment:
It would be preferable to omit the modifiers that are default. Without checking I think all of ones specified here are.

Reply:
So the main issue that I had reading the documentation was trying to understand how to apply modifiers at all. The state diagrams were initially confusing but I was able to figure it out by iterating on it. Maybe I should give an example that uses a different format to show how modifiers work? The main thing I'd like to see demonstrated is >1 modifier for a given component.

Reply:
Ah, I gotcha. Yes, a separate example would probably be clearer.

Thank you for the additions, by the way! Every bit helps.